controllers/pipelines/runconfiguration_controller.go

```diff
@@ -324,19 +330,18 @@ func findOwnedRuns(ctx context.Context, cli client.Reader, runConfiguration *pip
 }
 
 func (r *RunConfigurationReconciler) constructRunForRunConfiguration(provider string, runConfiguration *pipelinesv1.RunConfiguration) (*pipelinesv1.Run, error) {
+	spec := runConfiguration.Spec.Run
```

Comment:
This would populate artifacts and runtime parameters (that would be filled possibly from dependencies by the run controller)... correct?

Reply:
runtime parameters in one-off runs are handled the same as in RunConfigurations (Resolution in the respective controller). We only pre-resolve them for RunSchedules at the moment. This change doesn't change this behaviour to what we had [before](https://github.com/sky-uk/kfp-operator/pull/291/files#diff-0a4eb9edb6dea35ffe018762cb542cec35b15b47e9151a04885cb5d7fc382a54L337).
So yes, this would be filled by the run controller